src/components/HdTileSelect.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <section class="field tile-select">
```

Comment:
Is `field` needed? Right now we have two block elements on one tag.

Reply:
It comes from here: https://github.com/homeday-de/homeday-blocks/blob/develop/src/styles/inputs.scss#L9

Reply:
In my opinion, we should use `<div>` tags instead of `<section>` element.

> Each `<section>` should be identified, typically by including a heading (`<h1>-<h6>` element) as a child of the `<section>` element.

> Do not use the `<section>` element as a generic container; this is what `<div>` is for, especially when the sectioning is only for styling purposes. A rule of thumb is that a section should logically appear in the outline of a document.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

Reply:
Done here: https://github.com/homeday-de/homeday-blocks/pull/465/commits/b0c0313c2555b0b58b0fb6f1db2a06f58d7692dd